### ISSUE TITLE
Improve the question marks detail

### ISSUE THREAD

**Issue by GriffGreen** [1Hive/time-lock-app#79]
The dandelion deployment looks great! but i'm a little confused about this app and how to deploy it properly..

I don't really understand the purpose of the time lock app... it could use an explainer some where... Am I locking some tokens? what for? What address should i put in to this field? 

The explainers don't make much sense without the context of what we are locking tokens for..

<img width="575" alt="Screen Shot 2019-11-13 at 5 14 54 PM" src="https://user-images.githubusercontent.com/14945613/68761220-37b04e00-0646-11ea-86e7-a86cd3c009f7.png">

and i hope this is a place holder ;-)

I am guessing, cause I have played the moloch game, that this is the functionality of 10 WETH for submitting a proposal... but if that is true, I only guessed it cause i have actually submitted proposals to moloch and had to pay it.

**Comment by lkngtn** [comment]
Perhaps adding a brief explanation of what the Time Lock apps purpose is above the configuration would help. Perhaps something along these lines:

> The time lock app is used discourage spammy, low-quality proposal submission by requiring users to lock tokens for a period of time whenever they submit a proposal. The amount and duration of each lock increases the more simultaneously active locks the user has accrued. 

Do you think that would be sufficient contextual information to make the rest of the configuration more intuitive?

For the lock duration popup, agree that description is not ideal. How does the following sound:

> Lock Duration determines the minimum amount of time the configured tokens must be locked in order to perform an action.

**Comment by rperez89** [comment]
<img width="1680" alt="Screen Shot 2019-11-21 at 5 02 47 PM" src="https://user-images.githubusercontent.com/11763623/69372612-ca716e00-0c80-11ea-9a43-d93b0caf57d2.png">

<img width="856" alt="Screen Shot 2019-11-21 at 5 03 34 PM" src="https://user-images.githubusercontent.com/11763623/69372662-e37a1f00-0c80-11ea-905b-7534d18bdfbd.png">
<img width="1007" alt="Screen Shot 2019-11-21 at 5 04 04 PM" src="https://user-images.githubusercontent.com/11763623/69372701-fb51a300-0c80-11ea-929b-3114038dcbde.png">

@GriffGreen @lkngtn how do you feel with those changes now?

**Comment by GriffGreen** [comment]
This looks WAY better! I think I could understand it! 

I assume the time lock  app could be applied to many general applications and thats why it says "perform an action" instead of "submit a proposal to vote"

**Comment by rperez89** [comment]
> This looks WAY better! I think I could understand it!
> 
> I assume the time lock app could be applied to many general applications and thats why it says "perform an action" instead of "submit a proposal to vote"
yeah, the idea is to also be able to use this app for any action that might require to lock some tokens

**Comment by lkngtn** [comment]
Not sure what happened to this, but it seems that this info component was removed from the onboarding at some point. We should put it back in.
